inst/examples/ex01-basic/parmesan/inputs.yaml

```diff
@@ -7,6 +7,9 @@ bins:
     max: 50
     value: 10
     step: 1
+  input_info:
+    icon:
```

Comment:
do not leave blank, even when you are providing a default when NULL users will use the examples to learn how to change it. Use icon: info-circle